lib/chef-dk/helpers.rb

```diff
@@ -113,16 +131,37 @@ def git_windows_bin_dir
       @git_windows_bin_dir ||= File.expand_path(File.join(omnibus_root, "embedded", "git", "usr", "bin"))
     end
 
+    def habitat_embedded_bin_dir
+      @habitat_embedded_bin_dir ||= ENV["HAB_WS_EMBEDDED_BIN_DIR"]
+    end
+
+    def habitat_bin_dir
+      @habitat_bin_dir ||= ENV["HAB_WS_BIN_DIR"]
+    end
```

Comment:
ideally this could be collapsed if the ruby + binstubs are in the same directory in habitat?  then you'd only need one path here, and only jam the one path into the user's PATH.